xilem/src/view/flex.rs

```diff
@@ -36,7 +36,7 @@ use crate::{AnyWidgetView, Pod, PropertyTuple as _, ViewCtx, WidgetView};
 /// }
 ///
 /// fn app_logic(data: &mut i32) -> impl WidgetView<i32> + use<> {
-///     flex((
+///     flex(Axis::Horizontal, (
```

Comment:
Might be worth mentioning the specialized components `column`,`row` as an alternative to using this, somewhere within this doc-comment.